pkg/controllers/networkpolicy/networkpolicy.go

```diff
@@ -362,6 +377,12 @@ func (s *securityGroupImpl) deleteImpl(c cloudSecurityGroup, membershipOnly bool
 				s.deletePending = true
 				return nil
 			}
+		} else {
+			var err error
+			nps, err = r.networkPolicyIndexer.ByIndex(networkPolicyIndexerByAppliedToGrp, s.id.Name)
+			if err != nil {
+				return fmt.Errorf("get networkpolicy indexer with index=%v, name=%v: %w", networkPolicyIndexerByAppliedToGrp, s.id.Name, err)
```

Comment:
nit: 
```suggestion
				return fmt.Errorf("get networkpolicy indexer with index=%v, name=%v: %v", networkPolicyIndexerByAppliedToGrp, s.id.Name, err)
```

Reply:
Done

Reply:
Actually %w can and should be used for errors in format.
https://stackoverflow.com/questions/61283248/format-errors-in-go-s-v-or-w